client/src/redux/reducers/tickets.reducers.js

```diff
@@ -147,26 +147,56 @@ export const initialTicketCreateState = {
 
 export function ticketCreate(state = initialTicketCreateState, action) {
   switch (action.type) {
-    case 'TICKETS_CREATE_INIT':
+    case 'TICKET_CREATE_INIT':
```

Comment:
@jmartinez-tacc , fyi: I changed this action name:  TICKETS_CREATE_INIT -> TICKET_CREATE_INIT